lib/src/util.h

```diff
@@ -161,8 +165,8 @@ static inline void *mdup(const void *src, size_t n)
  * @return The number of characters written to the buffer, or -1 if an
  *         error occurs (errno will be set).
  */
-ssize_t snprintf_realloc(char ** restrict buf, size_t *bufcap,
-        const char * restrict format, ...) __cypherlang_format(3, 4);
```

Comment:
Is `restrict` not supported in WIN32?

Reply:
Yes it is, the keyword is slightly different: `__restrict`.
I will put back this code and add a `#define restrict __restrict` in `.cmake/config.h.in`.